go.mod

```diff
@@ -15,11 +15,14 @@ require (
 	github.com/go-playground/universal-translator v0.16.0 // indirect
 	github.com/golang-migrate/migrate v3.5.4+incompatible // indirect
 	github.com/golang/mock v1.3.1
+	github.com/goware/modvendor v0.0.0-20190516042800-ce72b408a8fe // indirect
```

Comment:
same as for bindata

Reply:
The same case, missing `go mod tidy` after installing `github.com/goware/modvendor`.